From a0f30a5c6002173c6f2ffa40247c68a5a94477b4 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <luca.boccassi@gmail.com>
Date: Tue, 10 Dec 2024 11:27:41 +0000
Subject: [PATCH] Update hwdb

ninja -C build update-hwdb

(cherry picked from commit 0d740f4bc0f03b773264ef75eaf1233ffc89c5b2)

Resolves: RHEL-49810
---
 hwdb.d/20-OUI.hwdb               | 322 ++++++++++++++++-
 hwdb.d/20-acpi-vendor.hwdb.patch |   4 +-
 hwdb.d/20-usb-vendor-model.hwdb  |   2 +-
 hwdb.d/ma-large.txt              | 577 ++++++++++++++++++++++++++-----
 hwdb.d/ma-medium.txt             | 124 ++++++-
 hwdb.d/ma-small.txt              | 142 +++++++-
 hwdb.d/usb.ids                   |   6 +-
 7 files changed, 1058 insertions(+), 119 deletions(-)

diff --git a/hwdb.d/20-OUI.hwdb b/hwdb.d/20-OUI.hwdb
index 7011126f01..a692317fd0 100644
--- a/hwdb.d/20-OUI.hwdb
+++ b/hwdb.d/20-OUI.hwdb
@@ -19893,7 +19893,7 @@ OUI:001A00*
  ID_OUI_FROM_DATABASE=MATRIX INC.
 
 OUI:001A01*
- ID_OUI_FROM_DATABASE=Smiths Medical
+ ID_OUI_FROM_DATABASE=ICU Medical, Inc.
 
 OUI:001A02*
  ID_OUI_FROM_DATABASE=SECURE CARE PRODUCTS, INC
@@ -25908,7 +25908,7 @@ OUI:00210D*
  ID_OUI_FROM_DATABASE=SAMSIN INNOTEC
 
 OUI:00210E*
- ID_OUI_FROM_DATABASE=Orpak Systems L.T.D.
+ ID_OUI_FROM_DATABASE=Gilbarco Inc.
 
 OUI:00210F*
  ID_OUI_FROM_DATABASE=Cernium Corp
@@ -40649,6 +40649,9 @@ OUI:04BF6D*
 OUI:04BFA8*
  ID_OUI_FROM_DATABASE=ISB Corporation
 
+OUI:04BFD5*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:04C05B*
  ID_OUI_FROM_DATABASE=Tigo Energy
 
@@ -42218,6 +42221,9 @@ OUI:0887C6*
 OUI:0887C7*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
+OUI:088AF1*
+ ID_OUI_FROM_DATABASE=MERCUSYS TECHNOLOGIES CO., LTD.
+
 OUI:088BC8*
  ID_OUI_FROM_DATABASE=Google, Inc.
 
@@ -43718,6 +43724,9 @@ OUI:0CA402*
 OUI:0CA42A*
  ID_OUI_FROM_DATABASE=OB Telecom Electronic Technology Co., Ltd
 
+OUI:0CA64C*
+ ID_OUI_FROM_DATABASE=Hangzhou Ezviz Software Co.,Ltd.
+
 OUI:0CA694*
  ID_OUI_FROM_DATABASE=Sunitec Enterprise Co.,Ltd
 
@@ -45275,6 +45284,9 @@ OUI:10B9F7*
 OUI:10B9FE*
  ID_OUI_FROM_DATABASE=Lika srl
 
+OUI:10BA1A*
+ ID_OUI_FROM_DATABASE=SHENZHEN IK WORLD Technology Co., Ltd
+
 OUI:10BAA5*
  ID_OUI_FROM_DATABASE=GANA I&C CO., LTD
 
@@ -45353,6 +45365,9 @@ OUI:10C67E*
 OUI:10C6FC*
  ID_OUI_FROM_DATABASE=Garmin International
 
+OUI:10C735*
+ ID_OUI_FROM_DATABASE=Microsoft Corporation
+
 OUI:10C73F*
  ID_OUI_FROM_DATABASE=Midas Klark Teknik Ltd
 
@@ -45662,6 +45677,9 @@ OUI:140708*
 OUI:1407E0*
  ID_OUI_FROM_DATABASE=Abrantix AG
 
+OUI:140805*
+ ID_OUI_FROM_DATABASE=SKY UK LIMITED
+
 OUI:1409B4*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -49808,6 +49826,9 @@ OUI:1CF64C*
 OUI:1CF8D0*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:1CF9D5*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:1CFA68*
  ID_OUI_FROM_DATABASE=TP-LINK TECHNOLOGIES CO.,LTD.
 
@@ -50147,6 +50168,9 @@ OUI:2032C6*
 OUI:2034FB*
  ID_OUI_FROM_DATABASE=Xiaomi Communications Co Ltd
 
+OUI:203543*
+ ID_OUI_FROM_DATABASE=Sagemcom Broadband SAS
+
 OUI:203626*
  ID_OUI_FROM_DATABASE=TP-Link Corporation Limited
 
@@ -55148,6 +55172,9 @@ OUI:2CDFE6*
 OUI:2CE032*
  ID_OUI_FROM_DATABASE=TCL King Electrical Appliances(Huizhou)Co.,Ltd
 
+OUI:2CE099*
+ ID_OUI_FROM_DATABASE=FUJIAN STAR-NET COMMUNICATION CO.,LTD
+
 OUI:2CE2A8*
  ID_OUI_FROM_DATABASE=DeviceDesign
 
@@ -57182,6 +57209,9 @@ OUI:3464A9*
 OUI:346679*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:346691*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:3466EA*
  ID_OUI_FROM_DATABASE=VERTU INTERNATIONAL CORPORATION LIMITED
 
@@ -57380,6 +57410,9 @@ OUI:348C5E*
 OUI:348D52*
  ID_OUI_FROM_DATABASE=Sichuan Tianyi Comheart Telecom Co.,LTD
 
+OUI:348E89*
+ ID_OUI_FROM_DATABASE=Hangzhou BroadLink Technology Co., Ltd
+
 OUI:348F27*
  ID_OUI_FROM_DATABASE=Ruckus Wireless
 
@@ -57794,6 +57827,9 @@ OUI:34D2C4*
 OUI:34D4E3*
  ID_OUI_FROM_DATABASE=Atom Power, Inc.
 
+OUI:34D509*
+ ID_OUI_FROM_DATABASE=Genexis B.V.
+
 OUI:34D693*
  ID_OUI_FROM_DATABASE=Huawei Device Co., Ltd.
 
@@ -59159,6 +59195,9 @@ OUI:38C0EA*
 OUI:38C2BA*
  ID_OUI_FROM_DATABASE=CCTV NEOTECH
 
+OUI:38C43A*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:38C4E8*
  ID_OUI_FROM_DATABASE=NSS Sp. z o.o.
 
@@ -59225,6 +59264,9 @@ OUI:38D57A*
 OUI:38D620*
  ID_OUI_FROM_DATABASE=Limidea Concept Pte. Ltd.
 
+OUI:38D6E0*
+ ID_OUI_FROM_DATABASE=TOPDON TECHNOLOGY Co.,Ltd.
+
 OUI:38D7CA*
  ID_OUI_FROM_DATABASE=7HUGS LABS
 
@@ -60551,6 +60593,9 @@ OUI:3CBBFD*
 OUI:3CBCD0*
  ID_OUI_FROM_DATABASE=zte corporation
 
+OUI:3CBD14*
+ ID_OUI_FROM_DATABASE=Diebold Nixdorf
+
 OUI:3CBD3E*
  ID_OUI_FROM_DATABASE=Beijing Xiaomi Electronics Co., Ltd.
 
@@ -63020,6 +63065,9 @@ OUI:449F7F*
 OUI:44A038*
  ID_OUI_FROM_DATABASE=Huawei Device Co., Ltd.
 
+OUI:44A10E*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:44A191*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -66461,6 +66509,9 @@ OUI:50131D*
 OUI:501395*
  ID_OUI_FROM_DATABASE=Sichuan AI-Link Technology Co., Ltd.
 
+OUI:50139D*
+ ID_OUI_FROM_DATABASE=Sierra Wireless, ULC
+
 OUI:501408*
  ID_OUI_FROM_DATABASE=AiNET
 
@@ -67916,6 +67967,9 @@ OUI:540A77*
 OUI:540A8A*
  ID_OUI_FROM_DATABASE=Jlztlink Industry(ShenZhen)Co.,Ltd.
 
+OUI:540BB6*
+ ID_OUI_FROM_DATABASE=Variscite LTD
+
 OUI:540DF9*
  ID_OUI_FROM_DATABASE=Huawei Device Co., Ltd.
 
@@ -68312,6 +68366,9 @@ OUI:5466F9*
 OUI:546706*
  ID_OUI_FROM_DATABASE=GUANGDONG OPPO MOBILE TELECOMMUNICATIONS CORP.,LTD
 
+OUI:546749*
+ ID_OUI_FROM_DATABASE=Google, Inc.
+
 OUI:546751*
  ID_OUI_FROM_DATABASE=Compal Broadband Networks, Inc.
 
@@ -70278,7 +70335,7 @@ OUI:58FCC6*
  ID_OUI_FROM_DATABASE=TOZO INC
 
 OUI:58FCC8*
- ID_OUI_FROM_DATABASE=LenelS2 Carrier
+ ID_OUI_FROM_DATABASE=Honeywell
 
 OUI:58FCDB0*
  ID_OUI_FROM_DATABASE=Spang Power Electronics
@@ -71183,6 +71240,9 @@ OUI:5C9AA1*
 OUI:5C9AD8*
  ID_OUI_FROM_DATABASE=FUJITSU LIMITED
 
+OUI:5C9BA6*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:5CA06C*
  ID_OUI_FROM_DATABASE=Realme Chongqing Mobile Telecommunications Corp.,Ltd.
 
@@ -71246,6 +71306,9 @@ OUI:5CAC4C*
 OUI:5CAD76*
  ID_OUI_FROM_DATABASE=Shenzhen TCL New Technology Co., Ltd
 
+OUI:5CADBA*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:5CADCF*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
@@ -72044,6 +72107,9 @@ OUI:603D26*
 OUI:603D29*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:603D61*
+ ID_OUI_FROM_DATABASE=Private
+
 OUI:603E5F*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
@@ -73055,6 +73121,9 @@ OUI:6400F1*
 OUI:6401FB*
  ID_OUI_FROM_DATABASE=Landis+Gyr GmbH
 
+OUI:64028F*
+ ID_OUI_FROM_DATABASE=Silicon Laboratories
+
 OUI:6402CB*
  ID_OUI_FROM_DATABASE=Commscope
 
@@ -73397,6 +73466,9 @@ OUI:644214*
 OUI:644346*
  ID_OUI_FROM_DATABASE=GuangDong Quick Network Computer CO.,LTD
 
+OUI:64447B*
+ ID_OUI_FROM_DATABASE=vivo Mobile Communication Co., Ltd.
+
 OUI:6444D5*
  ID_OUI_FROM_DATABASE=TD Tech
 
@@ -74732,6 +74804,9 @@ OUI:6865B7*
 OUI:686725*
  ID_OUI_FROM_DATABASE=Espressif Inc.
 
+OUI:6867C7*
+ ID_OUI_FROM_DATABASE=Sagemcom Broadband SAS
+
 OUI:68692E*
  ID_OUI_FROM_DATABASE=Zycoo Co.,Ltd
 
@@ -75521,6 +75596,9 @@ OUI:6C0460*
 OUI:6C047A*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:6C05D3*
+ ID_OUI_FROM_DATABASE=CHINA DRAGON TECHNOLOGY LIMITED
+
 OUI:6C05D5*
  ID_OUI_FROM_DATABASE=Ethertronics Inc
 
@@ -76937,6 +77015,9 @@ OUI:7011AE*
 OUI:701301*
  ID_OUI_FROM_DATABASE=Vantiva - Connected Home
 
+OUI:701384*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:701404*
  ID_OUI_FROM_DATABASE=Limited Liability Company
 
@@ -90386,6 +90467,9 @@ OUI:70EB74*
 OUI:70ECE4*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
+OUI:70EDFA*
+ ID_OUI_FROM_DATABASE=imperix Ltd
+
 OUI:70EE50*
  ID_OUI_FROM_DATABASE=Netatmo
 
@@ -91220,6 +91304,9 @@ OUI:747548*
 OUI:7475DF*
  ID_OUI_FROM_DATABASE=TECLINK
 
+OUI:74761F*
+ ID_OUI_FROM_DATABASE=Microsoft Corporation
+
 OUI:74765B*
  ID_OUI_FROM_DATABASE=Quectel Wireless Solutions Co.,Ltd.
 
@@ -91961,6 +92048,9 @@ OUI:74F91A*
 OUI:74F9CA*
  ID_OUI_FROM_DATABASE=Nintendo Co.,Ltd
 
+OUI:74FB17*
+ ID_OUI_FROM_DATABASE=Qingdao Goertek  Horizons Tecnology  Co.,LTD
+
 OUI:74FDA0*
  ID_OUI_FROM_DATABASE=Compupal (Group) Corporation
 
@@ -92150,6 +92240,9 @@ OUI:781C1E*
 OUI:781C5A*
  ID_OUI_FROM_DATABASE=SHARP Corporation
 
+OUI:781C9D*
+ ID_OUI_FROM_DATABASE=Silicon Laboratories
+
 OUI:781D4A*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -92375,6 +92468,9 @@ OUI:783EA1*
 OUI:783F15*
  ID_OUI_FROM_DATABASE=EasySYNC Ltd.
 
+OUI:783F4D*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:7840E4*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -94553,6 +94649,9 @@ OUI:7CB59B*
 OUI:7CB59F*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:7CB68D*
+ ID_OUI_FROM_DATABASE=Mist Systems, Inc.
+
 OUI:7CB733*
  ID_OUI_FROM_DATABASE=ASKEY COMPUTER CORP
 
@@ -94751,6 +94850,9 @@ OUI:7CC8D0*
 OUI:7CC8D7*
  ID_OUI_FROM_DATABASE=Damalisk
 
+OUI:7CC8DF*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:7CC926*
  ID_OUI_FROM_DATABASE=Wuhan GreeNet Information Service Co.,Ltd.
 
@@ -95756,6 +95858,9 @@ OUI:808287*
 OUI:8082F5*
  ID_OUI_FROM_DATABASE=STMicrolectronics International NV
 
+OUI:8083F6*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:8084A9*
  ID_OUI_FROM_DATABASE=oshkosh Corporation
 
@@ -95861,6 +95966,9 @@ OUI:809FF5*
 OUI:80A036*
  ID_OUI_FROM_DATABASE=Shanghai MXCHIP Information Technology Co., Ltd.
 
+OUI:80A197*
+ ID_OUI_FROM_DATABASE=u-blox AG
+
 OUI:80A1AB*
  ID_OUI_FROM_DATABASE=Intellisis
 
@@ -96284,6 +96392,9 @@ OUI:80F25E*
 OUI:80F3EF*
  ID_OUI_FROM_DATABASE=Meta Platforms Technologies, LLC
 
+OUI:80F416*
+ ID_OUI_FROM_DATABASE=Chipsea Technologies (Shenzhen) Corp.
+
 OUI:80F503*
  ID_OUI_FROM_DATABASE=Commscope
 
@@ -96545,6 +96656,9 @@ OUI:842859*
 OUI:84285A*
  ID_OUI_FROM_DATABASE=Saffron Solutions Inc
 
+OUI:8428D6*
+ ID_OUI_FROM_DATABASE=SERVERCOM (INDIA) PRIVATE LIMITED
+
 OUI:842914*
  ID_OUI_FROM_DATABASE=EMPORIA TELECOM Produktions- und VertriebsgesmbH & Co KG
 
@@ -97118,6 +97232,9 @@ OUI:84948C*
 OUI:849681*
  ID_OUI_FROM_DATABASE=Cathay Communication Co.,Ltd
 
+OUI:849690*
+ ID_OUI_FROM_DATABASE=Murata Manufacturing Co., Ltd.
+
 OUI:8496D8*
  ID_OUI_FROM_DATABASE=Commscope
 
@@ -97187,6 +97304,9 @@ OUI:84A783*
 OUI:84A788*
  ID_OUI_FROM_DATABASE=Perples
 
+OUI:84A824*
+ ID_OUI_FROM_DATABASE=Google, Inc.
+
 OUI:84A8E4*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -97580,6 +97700,9 @@ OUI:84EA97*
 OUI:84EA99*
  ID_OUI_FROM_DATABASE=Vieworks
 
+OUI:84EAD2*
+ ID_OUI_FROM_DATABASE=KOGANEI CORPORATION
+
 OUI:84EAED*
  ID_OUI_FROM_DATABASE=Roku, Inc
 
@@ -98057,6 +98180,9 @@ OUI:8850F6*
 OUI:88517A*
  ID_OUI_FROM_DATABASE=HMD Global Oy
 
+OUI:8851F2*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:8851FB*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
@@ -99491,6 +99617,9 @@ OUI:8C1F64085*
 OUI:8C1F64086*
  ID_OUI_FROM_DATABASE=WEPTECH elektronik GmbH
 
+OUI:8C1F64089*
+ ID_OUI_FROM_DATABASE=ANSER-NET CO ., LTD
+
 OUI:8C1F6408B*
  ID_OUI_FROM_DATABASE=Shanghai Shenxu Technology Co., Ltd
 
@@ -99671,6 +99800,9 @@ OUI:8C1F640FA*
 OUI:8C1F640FE*
  ID_OUI_FROM_DATABASE=Indra Heera Technology LLP
 
+OUI:8C1F640FF*
+ ID_OUI_FROM_DATABASE=Pneumax Spa
+
 OUI:8C1F64101*
  ID_OUI_FROM_DATABASE=ASW-ATI Srl
 
@@ -99839,6 +99971,9 @@ OUI:8C1F6416E*
 OUI:8C1F64170*
  ID_OUI_FROM_DATABASE=Fracarro Radioindustrie Srl
 
+OUI:8C1F64174*
+ ID_OUI_FROM_DATABASE=KST technology
+
 OUI:8C1F64175*
  ID_OUI_FROM_DATABASE=Wuhan YiValley Opto-electric technology Co.,Ltd
 
@@ -100109,6 +100244,9 @@ OUI:8C1F64246*
 OUI:8C1F64247*
  ID_OUI_FROM_DATABASE=Dadhwal Weighing Instrument Repairing Works
 
+OUI:8C1F64249*
+ ID_OUI_FROM_DATABASE=TEX COMPUTER SRL
+
 OUI:8C1F6424C*
  ID_OUI_FROM_DATABASE=Shenzhen Link-All Technolgy Co., Ltd
 
@@ -100337,6 +100475,9 @@ OUI:8C1F642E5*
 OUI:8C1F642E8*
  ID_OUI_FROM_DATABASE=Sonora Network Solutions
 
+OUI:8C1F642EC*
+ ID_OUI_FROM_DATABASE=HD Vision Systems GmbH
+
 OUI:8C1F642EE*
  ID_OUI_FROM_DATABASE=Qualitrol LLC
 
@@ -100550,6 +100691,9 @@ OUI:8C1F6436F*
 OUI:8C1F64370*
  ID_OUI_FROM_DATABASE=WOLF Advanced Technology
 
+OUI:8C1F64371*
+ ID_OUI_FROM_DATABASE=LIMAB AB
+
 OUI:8C1F64372*
  ID_OUI_FROM_DATABASE=WINK Streaming
 
@@ -100844,12 +100988,18 @@ OUI:8C1F64427*
 OUI:8C1F64429*
  ID_OUI_FROM_DATABASE=Abbott Diagnostics Technologies AS
 
+OUI:8C1F6442A*
+ ID_OUI_FROM_DATABASE=Atonarp Inc.
+
 OUI:8C1F6442B*
  ID_OUI_FROM_DATABASE=Gamber Johnson-LLC
 
 OUI:8C1F64432*
  ID_OUI_FROM_DATABASE=Rebel Systems
 
+OUI:8C1F64437*
+ ID_OUI_FROM_DATABASE=Gogo BA
+
 OUI:8C1F64438*
  ID_OUI_FROM_DATABASE=Integer.pl S.A.
 
@@ -101513,6 +101663,9 @@ OUI:8C1F645F6*
 OUI:8C1F645F7*
  ID_OUI_FROM_DATABASE=Eagle Harbor Technologies, Inc.
 
+OUI:8C1F645F9*
+ ID_OUI_FROM_DATABASE=KRONOTECH SRL
+
 OUI:8C1F645FA*
  ID_OUI_FROM_DATABASE=PolCam Systems Sp. z o.o.
 
@@ -101861,6 +102014,9 @@ OUI:8C1F646EA*
 OUI:8C1F646EC*
  ID_OUI_FROM_DATABASE=Bit Trade One, Ltd.
 
+OUI:8C1F646EE*
+ ID_OUI_FROM_DATABASE=Cortical Labs Pte Ltd
+
 OUI:8C1F646F4*
  ID_OUI_FROM_DATABASE=Elsist Srl
 
@@ -102173,6 +102329,9 @@ OUI:8C1F647BB*
 OUI:8C1F647BC*
  ID_OUI_FROM_DATABASE=GO development GmbH
 
+OUI:8C1F647C2*
+ ID_OUI_FROM_DATABASE=CTI Intl Solutions
+
 OUI:8C1F647C6*
  ID_OUI_FROM_DATABASE=Flextronics International Kft
 
@@ -102305,6 +102464,9 @@ OUI:8C1F64811*
 OUI:8C1F64813*
  ID_OUI_FROM_DATABASE=Pribusin Inc.
 
+OUI:8C1F64814*
+ ID_OUI_FROM_DATABASE=DTI SRL
+
 OUI:8C1F64816*
  ID_OUI_FROM_DATABASE=PalmSens BV
 
@@ -102362,6 +102524,9 @@ OUI:8C1F6483E*
 OUI:8C1F64842*
  ID_OUI_FROM_DATABASE=Potter Electric Signal Co. LLC
 
+OUI:8C1F64846*
+ ID_OUI_FROM_DATABASE=Fo Xie Optoelectronics Technology Co., Ltd
+
 OUI:8C1F64848*
  ID_OUI_FROM_DATABASE=Jena-Optronik GmbH
 
@@ -103109,6 +103274,9 @@ OUI:8C1F64A47*
 OUI:8C1F64A49*
  ID_OUI_FROM_DATABASE=Integer.pl S.A.
 
+OUI:8C1F64A4A*
+ ID_OUI_FROM_DATABASE=YUYAMA MFG Co.,Ltd
+
 OUI:8C1F64A4C*
  ID_OUI_FROM_DATABASE=Flextronics International Kft
 
@@ -103400,6 +103568,9 @@ OUI:8C1F64B13*
 OUI:8C1F64B14*
  ID_OUI_FROM_DATABASE=Murata Manufacturing CO., Ltd.
 
+OUI:8C1F64B17*
+ ID_OUI_FROM_DATABASE=DAT Informatics Pvt Ltd
+
 OUI:8C1F64B18*
  ID_OUI_FROM_DATABASE=Grossenbacher Systeme AG
 
@@ -103538,6 +103709,9 @@ OUI:8C1F64B7C*
 OUI:8C1F64B7D*
  ID_OUI_FROM_DATABASE=Scheurich GmbH
 
+OUI:8C1F64B81*
+ ID_OUI_FROM_DATABASE=Prolife Equipamentos Médicos Ltda.
+
 OUI:8C1F64B82*
  ID_OUI_FROM_DATABASE=Seed Core Co., LTD.
 
@@ -103661,6 +103835,9 @@ OUI:8C1F64BC6*
 OUI:8C1F64BC9*
  ID_OUI_FROM_DATABASE=GL TECH CO.,LTD
 
+OUI:8C1F64BCA*
+ ID_OUI_FROM_DATABASE=OPTOKON, a.s.
+
 OUI:8C1F64BCB*
  ID_OUI_FROM_DATABASE=A&T Corporation
 
@@ -104312,6 +104489,9 @@ OUI:8C1F64D8E*
 OUI:8C1F64D8F*
  ID_OUI_FROM_DATABASE=DEUTA-WERKE GmbH
 
+OUI:8C1F64D90*
+ ID_OUI_FROM_DATABASE=SMITEC S.p.A.
+
 OUI:8C1F64D92*
  ID_OUI_FROM_DATABASE=Mitsubishi Electric India Pvt. Ltd.
 
@@ -104474,6 +104654,9 @@ OUI:8C1F64E00*
 OUI:8C1F64E02*
  ID_OUI_FROM_DATABASE=ITS Teknik A/S
 
+OUI:8C1F64E04*
+ ID_OUI_FROM_DATABASE=新川センサテクノロジ株式会社
+
 OUI:8C1F64E09*
  ID_OUI_FROM_DATABASE=ENLESS WIRELESS
 
@@ -104531,6 +104714,9 @@ OUI:8C1F64E30*
 OUI:8C1F64E33*
  ID_OUI_FROM_DATABASE=Amiad Water Systems
 
+OUI:8C1F64E35*
+ ID_OUI_FROM_DATABASE=Horcery LLC
+
 OUI:8C1F64E3C*
  ID_OUI_FROM_DATABASE=Finotex Electronic Solutions PVT LTD
 
@@ -106097,6 +106283,9 @@ OUI:8CB0E9*
 OUI:8CB64F*
  ID_OUI_FROM_DATABASE=Cisco Systems, Inc
 
+OUI:8CB6C5*
+ ID_OUI_FROM_DATABASE=Mimosa Networks
+
 OUI:8CB7F7*
  ID_OUI_FROM_DATABASE=Shenzhen UniStrong Science & Technology Co., Ltd
 
@@ -107603,6 +107792,9 @@ OUI:90CD1F*
 OUI:90CDB6*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:90CDE8*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:90CEB8*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
@@ -111365,6 +111557,9 @@ OUI:9C77AA*
 OUI:9C79AC*
  ID_OUI_FROM_DATABASE=Suntec Software(Shanghai) Co., Ltd.
 
+OUI:9C79E3*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:9C7A03*
  ID_OUI_FROM_DATABASE=Ciena Corporation
 
@@ -112886,6 +113081,9 @@ OUI:A0999B*
 OUI:A09A5A*
  ID_OUI_FROM_DATABASE=Time Domain
 
+OUI:A09A8E*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:A09B12*
  ID_OUI_FROM_DATABASE=China Mobile IOT Company Limited
 
@@ -114041,6 +114239,12 @@ OUI:A4580FD*
 OUI:A4580FE*
  ID_OUI_FROM_DATABASE=Finetree Communications Inc
 
+OUI:A459D32*
+ ID_OUI_FROM_DATABASE=Shenzhen Welltech Cable Co., Ltd
+
+OUI:A459D3A*
+ ID_OUI_FROM_DATABASE=OREX SAI Inc.
+
 OUI:A45A1C*
  ID_OUI_FROM_DATABASE=smart-electronic GmbH
 
@@ -115289,6 +115493,9 @@ OUI:A85BB0*
 OUI:A85BB7*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
+OUI:A85BD1*
+ ID_OUI_FROM_DATABASE=INVENTEC CORPORATION
+
 OUI:A85BF3*
  ID_OUI_FROM_DATABASE=Audivo GmbH
 
@@ -117029,6 +117236,9 @@ OUI:ACC906*
 OUI:ACC935*
  ID_OUI_FROM_DATABASE=Ness Corporation
 
+OUI:ACCA0F*
+ ID_OUI_FROM_DATABASE=INDISR COMMUNICATION SERVICES AND TECHNOLOGIES INDIA
+
 OUI:ACCA54*
  ID_OUI_FROM_DATABASE=Telldus Technologies AB
 
@@ -117107,6 +117317,9 @@ OUI:ACD829*
 OUI:ACD8A7*
  ID_OUI_FROM_DATABASE=BELLDESIGN Inc.
 
+OUI:ACD929*
+ ID_OUI_FROM_DATABASE=Quectel Wireless Solutions Co.,Ltd.
+
 OUI:ACD9D6*
  ID_OUI_FROM_DATABASE=tci GmbH
 
@@ -117395,6 +117608,9 @@ OUI:B01656*
 OUI:B01743*
  ID_OUI_FROM_DATABASE=EDISON GLOBAL CIRCUITS LLC
 
+OUI:B01831*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:B01886*
  ID_OUI_FROM_DATABASE=SmarDTV Corporation
 
@@ -121793,6 +122009,9 @@ OUI:BC8529*
 OUI:BC8556*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:BC85D0*
+ ID_OUI_FROM_DATABASE=SKY UK LIMITED
+
 OUI:BC87FA*
  ID_OUI_FROM_DATABASE=Bose Corporation
 
@@ -121946,6 +122165,9 @@ OUI:BC9E2C*
 OUI:BC9EBB*
  ID_OUI_FROM_DATABASE=Nintendo Co.,Ltd
 
+OUI:BC9F58*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:BC9FE4*
  ID_OUI_FROM_DATABASE=Hewlett Packard Enterprise
 
@@ -125597,6 +125819,9 @@ OUI:C84782*
 OUI:C8478C*
  ID_OUI_FROM_DATABASE=Beken Corporation
 
+OUI:C84805*
+ ID_OUI_FROM_DATABASE=Nintendo Co.,Ltd
+
 OUI:C848F5*
  ID_OUI_FROM_DATABASE=MEDISON Xray Co., Ltd
 
@@ -127622,6 +127847,9 @@ OUI:CCB0DA*
 OUI:CCB11A*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:CCB148*
+ ID_OUI_FROM_DATABASE=Arcadyan Corporation
+
 OUI:CCB182*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -128570,6 +128798,9 @@ OUI:D04CC1*
 OUI:D04D2C*
  ID_OUI_FROM_DATABASE=Roku, Inc.
 
+OUI:D04D86*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:D04DC6*
  ID_OUI_FROM_DATABASE=Hewlett Packard Enterprise
 
@@ -128735,6 +128966,9 @@ OUI:D05F64D*
 OUI:D05F64E*
  ID_OUI_FROM_DATABASE=Montblanc-Simplo GmbH
 
+OUI:D05FAF*
+ ID_OUI_FROM_DATABASE=Shenzhen C-Data Technology Co., Ltd.
+
 OUI:D05FB8*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
@@ -129242,9 +129476,54 @@ OUI:D0A637*
 OUI:D0A9D3*
  ID_OUI_FROM_DATABASE=EM Microelectronic
 
+OUI:D0AA5F0*
+ ID_OUI_FROM_DATABASE=Hangzhou Jingtang Communication Technology Co.,Ltd.
+
+OUI:D0AA5F1*
+ ID_OUI_FROM_DATABASE=Xiaomi EV Technology Co., Ltd.
+
+OUI:D0AA5F2*
+ ID_OUI_FROM_DATABASE=ACCEL LAB ltd.
+
+OUI:D0AA5F3*
+ ID_OUI_FROM_DATABASE=Shenzhen Liandian Communication Technology Co.LTD
+
+OUI:D0AA5F4*
+ ID_OUI_FROM_DATABASE=KOMER
+
+OUI:D0AA5F5*
+ ID_OUI_FROM_DATABASE=深圳麦源实业有限公司
+
+OUI:D0AA5F6*
+ ID_OUI_FROM_DATABASE=SORDIN AB
+
+OUI:D0AA5F7*
+ ID_OUI_FROM_DATABASE=广东美的暖通设备有限公司
+
+OUI:D0AA5F8*
+ ID_OUI_FROM_DATABASE=WeBank Co., Ltd.
+
+OUI:D0AA5F9*
+ ID_OUI_FROM_DATABASE=HONGCI MONITOR INTERNATIONAL CO., LTD.
+
 OUI:D0AA5FA*
  ID_OUI_FROM_DATABASE=Indra Air Traffic Inc
 
+OUI:D0AA5FB*
+ ID_OUI_FROM_DATABASE=WARNER ELECTRONICS (I) PVT. LTD.
+
+OUI:D0AA5FC*
+ ID_OUI_FROM_DATABASE=Pacific Electronics
+
+OUI:D0AA5FD*
+ ID_OUI_FROM_DATABASE=Level Up Holding Co., Inc.
+
+OUI:D0AA5FE*
+ ID_OUI_FROM_DATABASE=MBX International Ltd.
+
+OUI:D0AB4A*
+ ID_OUI_FROM_DATABASE=Telink Micro LLC
+
 OUI:D0ABD5*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -131081,6 +131360,9 @@ OUI:D8052E*
 OUI:D806D1*
  ID_OUI_FROM_DATABASE=Honeywell Fire System (Shanghai) Co,. Ltd.
 
+OUI:D806F3*
+ ID_OUI_FROM_DATABASE=Arista Networks
+
 OUI:D807B6*
  ID_OUI_FROM_DATABASE=TP-LINK TECHNOLOGIES CO.,LTD.
 
@@ -131129,6 +131411,9 @@ OUI:D80E29*
 OUI:D80F99*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
 
+OUI:D80FB5*
+ ID_OUI_FROM_DATABASE=SHENZHEN ULTRAEASY TECHNOLOGY CO LTD
+
 OUI:D81068*
  ID_OUI_FROM_DATABASE=Murata Manufacturing Co., Ltd.
 
@@ -132440,6 +132725,9 @@ OUI:DC3130*
 OUI:DC31D1*
  ID_OUI_FROM_DATABASE=vivo Mobile Communication Co., Ltd.
 
+OUI:DC3262*
+ ID_OUI_FROM_DATABASE=Nanjing Qinheng Microelectronics Co., Ltd.
+
 OUI:DC330D*
  ID_OUI_FROM_DATABASE=QING DAO HAIER TELECOM CO.,LTD.
 
@@ -132731,6 +133019,9 @@ OUI:DC54D7*
 OUI:DC5583*
  ID_OUI_FROM_DATABASE=GUANGDONG OPPO MOBILE TELECOMMUNICATIONS CORP.,LTD
 
+OUI:DC567B*
+ ID_OUI_FROM_DATABASE=CLOUD NETWORK TECHNOLOGY SINGAPORE PTE. LTD.
+
 OUI:DC56E6*
  ID_OUI_FROM_DATABASE=Shenzhen Bococom Technology Co.,LTD
 
@@ -132896,6 +133187,9 @@ OUI:DC86D8*
 OUI:DC87CB*
  ID_OUI_FROM_DATABASE=Beijing Perfectek Technologies Co., Ltd.
 
+OUI:DC88A1*
+ ID_OUI_FROM_DATABASE=ITEL MOBILE LIMITED
+
 OUI:DC8983*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -133584,7 +133878,7 @@ OUI:E01C41*
  ID_OUI_FROM_DATABASE=Extreme Networks Headquarters
 
 OUI:E01CA7*
- ID_OUI_FROM_DATABASE=Arista Networks, Inc.
+ ID_OUI_FROM_DATABASE=Arista Networks
 
 OUI:E01CEE*
  ID_OUI_FROM_DATABASE=Bravo Tech, Inc.
@@ -135026,6 +135320,9 @@ OUI:E44122*
 OUI:E44164*
  ID_OUI_FROM_DATABASE=Nokia
 
+OUI:E441D4*
+ ID_OUI_FROM_DATABASE=vivo Mobile Communication Co., Ltd.
+
 OUI:E441E6*
  ID_OUI_FROM_DATABASE=Ottec Technology GmbH
 
@@ -135881,6 +136178,9 @@ OUI:E4FD8C*
 OUI:E4FDA1*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:E4FE43*
+ ID_OUI_FROM_DATABASE=Beijing Xiaomi Mobile Software Co., Ltd
+
 OUI:E4FED9*
  ID_OUI_FROM_DATABASE=EDMI Europe Ltd
 
@@ -137057,6 +137357,9 @@ OUI:E8E1E1*
 OUI:E8E1E2*
  ID_OUI_FROM_DATABASE=Energotest
 
+OUI:E8E49D*
+ ID_OUI_FROM_DATABASE=Nexthop Systems Inc.
+
 OUI:E8E5D6*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
@@ -138518,6 +138821,9 @@ OUI:ECFCC6*
 OUI:ECFE7E*
  ID_OUI_FROM_DATABASE=BlueRadios, Inc.
 
+OUI:ECFF3A*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:F0007F*
  ID_OUI_FROM_DATABASE=Janz - Contadores de Energia, SA
 
@@ -139673,6 +139979,9 @@ OUI:F0D9B2*
 OUI:F0DA7C*
  ID_OUI_FROM_DATABASE=RLH INDUSTRIES,INC.
 
+OUI:F0DB2A*
+ ID_OUI_FROM_DATABASE=LANNER ELECTRONICS, INC.
+
 OUI:F0DB30*
  ID_OUI_FROM_DATABASE=Yottabyte
 
@@ -140922,7 +141231,7 @@ OUI:F4B52F*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
 OUI:F4B549*
- ID_OUI_FROM_DATABASE=Xiamen Yeastar Information Technology Co., Ltd.
+ ID_OUI_FROM_DATABASE=Xiamen Yeastar Digital Technology Co., Ltd
 
 OUI:F4B5AA*
  ID_OUI_FROM_DATABASE=zte corporation
@@ -141041,6 +141350,9 @@ OUI:F4CAE7*
 OUI:F4CB52*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:F4CBE7*
+ ID_OUI_FROM_DATABASE=Apple, Inc.
+
 OUI:F4CC55*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
diff --git a/hwdb.d/20-acpi-vendor.hwdb.patch b/hwdb.d/20-acpi-vendor.hwdb.patch
index ced09a35b4..d95d550bc1 100644
--- a/hwdb.d/20-acpi-vendor.hwdb.patch
+++ b/hwdb.d/20-acpi-vendor.hwdb.patch
@@ -1,5 +1,5 @@
---- 20-acpi-vendor.hwdb.base	2024-11-27 14:50:06.308224101 +0000
-+++ 20-acpi-vendor.hwdb	2024-11-27 14:50:06.312224101 +0000
+--- 20-acpi-vendor.hwdb.base	2024-12-10 11:26:23.564234401 +0000
++++ 20-acpi-vendor.hwdb	2024-12-10 11:26:23.572234401 +0000
 @@ -3,6 +3,8 @@
  # Data imported from:
  #     https://uefi.org/uefi-pnp-export
diff --git a/hwdb.d/20-usb-vendor-model.hwdb b/hwdb.d/20-usb-vendor-model.hwdb
index 1b70bd9c89..1ffabba139 100644
--- a/hwdb.d/20-usb-vendor-model.hwdb
+++ b/hwdb.d/20-usb-vendor-model.hwdb
@@ -50568,7 +50568,7 @@ usb:v1209p0CBD*
  ID_MODEL_FROM_DATABASE=Andrzej Szombierski kuku.eu.org keyboard
 
 usb:v1209p0D32*
- ID_MODEL_FROM_DATABASE=ODrive Robotics ODrive v3
+ ID_MODEL_FROM_DATABASE=ODrive Robotics ODrive
 
 usb:v1209p1001*
  ID_MODEL_FROM_DATABASE=InterBiometrics Hub
diff --git a/hwdb.d/ma-large.txt b/hwdb.d/ma-large.txt
index f68bf88ee0..f8203cd6e8 100644
--- a/hwdb.d/ma-large.txt
+++ b/hwdb.d/ma-large.txt
@@ -26783,12 +26783,6 @@ B4ED54     (base 16)		Wohler Technologies
 				Hayward  CA  94544
 				US
 
-F8-DC-7A   (hex)		Variscite LTD
-F8DC7A     (base 16)		Variscite LTD
-				60 Medinat Hayehudim st.
-				Hertzelia    46722
-				IL
-
 B8-F7-32   (hex)		Aryaka Networks Inc
 B8F732     (base 16)		Aryaka Networks Inc
 				691 S. Milpitas Blvd., 
@@ -40532,12 +40526,6 @@ CC115A     (base 16)		Apple, Inc.
 				Taipei  Taiwan  112
 				TW
 
-58-FC-C8   (hex)		LenelS2 Carrier
-58FCC8     (base 16)		LenelS2 Carrier
-				1212 Pittsford Victor Rd
-				Pittsford  NY  14534
-				US
-
 18-95-78   (hex)		DENSO Corporation
 189578     (base 16)		DENSO Corporation
 				1-1, Showa-cho
@@ -43895,18 +43883,6 @@ BCF2E5     (base 16)		Powerful Devices
 				Piscataway  NJ  08554
 				US
 
-38-92-31   (hex)		TECNO MOBILE LIMITED
-389231     (base 16)		TECNO MOBILE LIMITED
-				ROOMS 05-15, 13A/F., SOUTH TOWER, WORLD FINANCE CENTRE, HARBOUR CITY, 17 CANTON ROAD, TSIM SHA TSUI, KOWLOON, HONG KONG
-				Hong Kong  Hong Kong  999077
-				HK
-
-58-8C-81   (hex)		Espressif Inc.
-588C81     (base 16)		Espressif Inc.
-				Room 204, Building 2, 690 Bibo Rd, Pudong New Area
-				Shanghai  Shanghai  201203
-				CN
-
 34-86-DA   (hex)		Honor Device Co., Ltd.
 3486DA     (base 16)		Honor Device Co., Ltd.
 				Suite 3401, Unit A, Building 6, Shum Yip Sky Park, No. 8089, Hongli West Road, Xiangmihu Street, Futian District 
@@ -43925,6 +43901,24 @@ F8C903     (base 16)		TP-LINK TECHNOLOGIES CO.,LTD.
 				Shenzhen  Guangdong  518057
 				CN
 
+38-92-31   (hex)		TECNO MOBILE LIMITED
+389231     (base 16)		TECNO MOBILE LIMITED
+				ROOMS 05-15, 13A/F., SOUTH TOWER, WORLD FINANCE CENTRE, HARBOUR CITY, 17 CANTON ROAD, TSIM SHA TSUI, KOWLOON, HONG KONG
+				Hong Kong  Hong Kong  999077
+				HK
+
+58-8C-81   (hex)		Espressif Inc.
+588C81     (base 16)		Espressif Inc.
+				Room 204, Building 2, 690 Bibo Rd, Pudong New Area
+				Shanghai  Shanghai  201203
+				CN
+
+6C-05-D3   (hex)		CHINA DRAGON TECHNOLOGY LIMITED
+6C05D3     (base 16)		CHINA DRAGON TECHNOLOGY LIMITED
+				B4 Building,No.3 First industrial Zone,Nanpu Road,Lao Community,Xinqian Street,Baoan District,Shenzhen,City
+				ShenZhen    518100
+				CN
+
 7C-8C-09   (hex)		Mellanox Technologies, Inc.
 7C8C09     (base 16)		Mellanox Technologies, Inc.
 				350 Oakmead Parkway, Suite 100 
@@ -43943,6 +43937,126 @@ F43392     (base 16)		Cisco Systems, Inc
 				San Jose  CA  94568
 				US
 
+84-28-D6   (hex)		SERVERCOM (INDIA) PRIVATE LIMITED
+8428D6     (base 16)		SERVERCOM (INDIA) PRIVATE LIMITED
+				E-43/1 OKHLA INDUSTRIAL AREA PHASE-II NEW DELHI SOUTH DELHI
+				NEW DELHI    NA
+				IN
+
+AC-D9-29   (hex)		Quectel Wireless Solutions Co.,Ltd.
+ACD929     (base 16)		Quectel Wireless Solutions Co.,Ltd.
+				7th Floor, Hongye Building, No.1801 Hongmei Road, Xuhui District
+				Shanghai    200233
+				CN
+
+10-C7-35   (hex)		Microsoft Corporation
+10C735     (base 16)		Microsoft Corporation
+				One Microsoft Way
+				REDMOND    98052
+				US
+
+CC-B1-48   (hex)		Arcadyan Corporation
+CCB148     (base 16)		Arcadyan Corporation
+				No.8, Sec.2, Guangfu Rd.
+				Hsinchu City  Hsinchu  30071
+				TW
+
+DC-56-7B   (hex)		CLOUD NETWORK TECHNOLOGY SINGAPORE PTE. LTD.
+DC567B     (base 16)		CLOUD NETWORK TECHNOLOGY SINGAPORE PTE. LTD.
+				B22 Building,NO.51 Tongle Road, Shajing Town, Jiangnan District, Nanning, Guangxi Province, China
+				Nanning  Guangxi  530007
+				CN
+
+34-8E-89   (hex)		Hangzhou BroadLink Technology Co., Ltd
+348E89     (base 16)		Hangzhou BroadLink Technology Co., Ltd
+				Room 101,1/F,Unit C,Building 1,No.57 Jiang'er Road,Changhe Street,Binjiang District
+				Hangzhou  ZheJiang  310052
+				CN
+
+64-02-8F   (hex)		Silicon Laboratories
+64028F     (base 16)		Silicon Laboratories
+				 400 West Cesar Chavez
+				Austin  TX  78701
+				US
+
+78-3F-4D   (hex)		Apple, Inc.
+783F4D     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+9C-79-E3   (hex)		Apple, Inc.
+9C79E3     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+88-51-F2   (hex)		Apple, Inc.
+8851F2     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+5C-AD-BA   (hex)		Apple, Inc.
+5CADBA     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+74-FB-17   (hex)		Qingdao Goertek  Horizons Tecnology  Co.,LTD
+74FB17     (base 16)		Qingdao Goertek  Horizons Tecnology  Co.,LTD
+				155 Zhuzhou,road
+				Qingdao  Shandong  266101
+				CN
+
+1C-F9-D5   (hex)		Apple, Inc.
+1CF9D5     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+54-67-49   (hex)		Google, Inc.
+546749     (base 16)		Google, Inc.
+				1600 Amphitheatre Parkway
+				Mountain View  CA  94043
+				US
+
+F8-DC-7A   (hex)		Variscite LTD
+F8DC7A     (base 16)		Variscite LTD
+				15 Ismargad St
+				Kiryat Gat    8202041
+				IL
+
+58-FC-C8   (hex)		Honeywell
+58FCC8     (base 16)		Honeywell
+				1212 Pittsford Victor Rd
+				Pittsford  NY  14534
+				US
+
+50-13-9D   (hex)		Sierra Wireless, ULC
+50139D     (base 16)		Sierra Wireless, ULC
+				13811 Wireless Way
+				Richmond  BC  V6V 3A4
+				CA
+
+BC-85-D0   (hex)		SKY UK LIMITED
+BC85D0     (base 16)		SKY UK LIMITED
+				Grant Way
+				Isleworth  Middlesex  TW7 5QD
+				GB
+
+14-08-05   (hex)		SKY UK LIMITED
+140805     (base 16)		SKY UK LIMITED
+				Grant Way
+				Isleworth  Middlesex  TW7 5QD
+				GB
+
+AC-CA-0F   (hex)		INDISR COMMUNICATION SERVICES AND TECHNOLOGIES INDIA
+ACCA0F     (base 16)		INDISR COMMUNICATION SERVICES AND TECHNOLOGIES INDIA
+				1207, Antriksh Bhawan 22, Kasturba GandhiMarg, New Delhi
+				New Delhi  Delhi  110001
+				IN
+
 00-01-30   (hex)		Extreme Networks Headquarters
 000130     (base 16)		Extreme Networks Headquarters
 				2121 RDU Center Drive 
@@ -87314,6 +87428,57 @@ A0C7D2     (base 16)		Cisco Systems, Inc
 				Hui Zhou  Guangdong  516025
 				CN
 
+04-BF-D5   (hex)		Apple, Inc.
+04BFD5     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+44-A1-0E   (hex)		Apple, Inc.
+44A10E     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+7C-C8-DF   (hex)		Apple, Inc.
+7CC8DF     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+E8-E4-9D   (hex)		Nexthop Systems Inc.
+E8E49D     (base 16)		Nexthop Systems Inc.
+				3110 Coronado Drive
+				Santa Clara  CA  95054
+				US
+
+80-83-F6   (hex)		Apple, Inc.
+8083F6     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+74-76-1F   (hex)		Microsoft Corporation
+74761F     (base 16)		Microsoft Corporation
+				One Microsoft Way
+				REDMOND    98052
+				US
+
+60-3D-61   (hex)		Private
+603D61     (base 16)		Private
+
+08-8A-F1   (hex)		MERCUSYS TECHNOLOGIES CO., LTD.
+088AF1     (base 16)		MERCUSYS TECHNOLOGIES CO., LTD.
+				3F,Zone B,Building R1,High-Tech Industrial Village,No.023 High-Tech South 4 Road,Nanshan,Shenzhen
+				Shenzhen  Guangdong  518057
+				CN
+
+10-BA-1A   (hex)		SHENZHEN IK WORLD Technology Co., Ltd
+10BA1A     (base 16)		SHENZHEN IK WORLD Technology Co., Ltd
+				291 Zhenxing Road, Guangming District, Shenzhen
+				Shenzhen  Guangdong  518000
+				CN
+
 6C-87-20   (hex)		New H3C Technologies Co., Ltd
 6C8720     (base 16)		New H3C Technologies Co., Ltd
 				466 Changhe Road, Binjiang District
@@ -129326,12 +129491,6 @@ F8F58C     (base 16)		Apple, Inc.
 				Cupertino  CA  95014
 				US
 
-E0-1C-A7   (hex)		Arista Networks, Inc.
-E01CA7     (base 16)		Arista Networks, Inc.
-				5453 Great America Parkway
-				Santa Clara    95054
-				US
-
 0C-85-E1   (hex)		Apple, Inc.
 0C85E1     (base 16)		Apple, Inc.
 				1 Infinite Loop
@@ -130991,17 +131150,11 @@ E45D39     (base 16)		Texas Instruments
 				Lippstadt    59552
 				DE
 
-10-A4-50   (hex)		Kwikset
-10A450     (base 16)		Kwikset
-				110 Sargent Dr.
-				New Haven  CT  06511
-				US
-
-4C-85-8A   (hex)		BUFFALO.INC
-4C858A     (base 16)		BUFFALO.INC
-				AKAMONDORI Bld.,30-20,Ohsu 3-chome,Naka-ku
-				Nagoya  Aichi Pref.  460-8315
-				JP
+E4-41-D4   (hex)		vivo Mobile Communication Co., Ltd.
+E441D4     (base 16)		vivo Mobile Communication Co., Ltd.
+				No.1, vivo Road, Chang'an
+				Dongguan  Guangdong  523860
+				CN
 
 08-3F-21   (hex)		Motorola Mobility LLC, a Lenovo Company
 083F21     (base 16)		Motorola Mobility LLC, a Lenovo Company
@@ -131009,12 +131162,102 @@ E45D39     (base 16)		Texas Instruments
 				Chicago  IL  60654
 				US
 
+64-44-7B   (hex)		vivo Mobile Communication Co., Ltd.
+64447B     (base 16)		vivo Mobile Communication Co., Ltd.
+				No.1, vivo Road, Chang'an
+				Dongguan  Guangdong  523860
+				CN
+
 98-E7-D5   (hex)		NXP Semiconductor (Tianjin) LTD.
 98E7D5     (base 16)		NXP Semiconductor (Tianjin) LTD.
 				No.15 Xinghua Avenue, Xiqing Economic Development Area
 				Tianjin    300385
 				CN
 
+4C-85-8A   (hex)		BUFFALO.INC
+4C858A     (base 16)		BUFFALO.INC
+				AKAMONDORI Bld.,30-20,Ohsu 3-chome,Naka-ku
+				Nagoya  Aichi Pref.  460-8315
+				JP
+
+10-A4-50   (hex)		Kwikset
+10A450     (base 16)		Kwikset
+				110 Sargent Dr.
+				New Haven  CT  06511
+				US
+
+C8-48-05   (hex)		Nintendo Co.,Ltd
+C84805     (base 16)		Nintendo Co.,Ltd
+				11-1 HOKOTATE-CHO KAMITOBA,MINAMI-KU
+				KYOTO  KYOTO  601-8501
+				JP
+
+80-A1-97   (hex)		u-blox AG
+80A197     (base 16)		u-blox AG
+				Zuercherstrasse, 68
+				Thalwil  Switzerland  CH-8800
+				CH
+
+8C-B6-C5   (hex)		Mimosa Networks
+8CB6C5     (base 16)		Mimosa Networks
+				3150 Coronado Dr
+				Santa Clara  CA  95054
+				US
+
+D8-0F-B5   (hex)		SHENZHEN ULTRAEASY TECHNOLOGY CO LTD
+D80FB5     (base 16)		SHENZHEN ULTRAEASY TECHNOLOGY CO LTD
+				22nd Floor，Block A，Huizhi R&D Center,NO.381 Xixiang Section,Guangshen Highway,Baoan District,Shenzhen City,Guangdong Province
+				SHENZHEN    518101
+				CN
+
+EC-FF-3A   (hex)		Apple, Inc.
+ECFF3A     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+70-13-84   (hex)		Apple, Inc.
+701384     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+84-A8-24   (hex)		Google, Inc.
+84A824     (base 16)		Google, Inc.
+				1600 Amphitheatre Parkway
+				Mountain View  CA  94043
+				US
+
+34-66-91   (hex)		Apple, Inc.
+346691     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+E0-1C-A7   (hex)		Arista Networks
+E01CA7     (base 16)		Arista Networks
+				5453 Great America Parkway
+				Santa Clara    95054
+				US
+
+84-EA-D2   (hex)		KOGANEI CORPORATION
+84EAD2     (base 16)		KOGANEI CORPORATION
+				3-11-28, Midori-cho
+				Koganei-shi  Tokyo  184-8533
+				JP
+
+F0-DB-2A   (hex)		LANNER ELECTRONICS, INC.
+F0DB2A     (base 16)		LANNER ELECTRONICS, INC.
+				7F, No. 173, Section 2, Datong Rd, Xizhi District
+				New Taipei City    22184
+				TW
+
+54-0B-B6   (hex)		Variscite LTD
+540BB6     (base 16)		Variscite LTD
+				15 Ismargad St
+				Kiryat Gat    8202041
+				IL
+
 B0-0C-9D   (hex)		Quectel Wireless Solutions Co.,Ltd.
 B00C9D     (base 16)		Quectel Wireless Solutions Co.,Ltd.
 				7th Floor, Hongye Building, No.1801 Hongmei Road, Xuhui District
@@ -155840,12 +156083,6 @@ CCF841     (base 16)		Lumewave
 				Seoul    150-871
 				KR
 
-F4-B5-49   (hex)		Xiamen Yeastar Information Technology Co., Ltd.
-F4B549     (base 16)		Xiamen Yeastar Information Technology Co., Ltd.
-				3/F, No. 46 Guanri Road, 2nd Software Park
-				Xiamen  Fujian  361006
-				CN
-
 68-5B-36   (hex)		POWERTECH INDUSTRIAL CO., LTD.
 685B36     (base 16)		POWERTECH INDUSTRIAL CO., LTD.
 				10F, No. 407, Chung Shan Rd., Sec 2,
@@ -160856,12 +161093,6 @@ A07332     (base 16)		Cashmaster International Limited
 				Blanchardstown  Dublin  15
 				IE
 
-00-1A-01   (hex)		Smiths Medical
-001A01     (base 16)		Smiths Medical
-				4350 Rivergreen Parkway
-				Duluth  GA  30096
-				US
-
 00-19-FE   (hex)		SHENZHEN SEECOMM TECHNOLOGY CO.,LTD.
 0019FE     (base 16)		SHENZHEN SEECOMM TECHNOLOGY CO.,LTD.
 				Room 609,404Yannan Road
@@ -175028,11 +175259,23 @@ D8F1D8     (base 16)		OI ELECTRIC CO.,LTD
 				Dallas  TX  75243
 				US
 
-18-DF-26   (hex)		INGRAM MICRO SERVICES
-18DF26     (base 16)		INGRAM MICRO SERVICES
-				100 CHEMIN DE BAILLOT
-				MONTAUBAN    82000
-				FR
+D0-85-43   (hex)		Cisco Systems, Inc
+D08543     (base 16)		Cisco Systems, Inc
+				80 West Tasman Drive
+				San Jose  CA  94568
+				US
+
+0C-7D-B0   (hex)		Motorola Mobility LLC, a Lenovo Company
+0C7DB0     (base 16)		Motorola Mobility LLC, a Lenovo Company
+				222 West Merchandise Mart Plaza
+				Chicago  IL  60654
+				US
+
+80-F4-16   (hex)		Chipsea Technologies (Shenzhen) Corp.
+80F416     (base 16)		Chipsea Technologies (Shenzhen) Corp.
+				3 / F, Block A, Building 2, Shenzhen Bay Innovation Technology Center, No.3156 keyuan South Road, Yuehai Street, Nanshan District, Shenzhen
+				Shenzhen  Guangdong  518000
+				CN
 
 80-79-33   (hex)		Aigentec Technology(Zhejiang) Co., Ltd.
 807933     (base 16)		Aigentec Technology(Zhejiang) Co., Ltd.
@@ -175040,22 +175283,112 @@ D8F1D8     (base 16)		OI ELECTRIC CO.,LTD
 				Hangzhou  Zhejiang Province  311200
 				CN
 
+D0-5F-AF   (hex)		Shenzhen C-Data Technology Co., Ltd.
+D05FAF     (base 16)		Shenzhen C-Data Technology Co., Ltd.
+				#201, Building A4, Nanshan Zhiyuan, No.1001, Xueyuan Avenue, Changyuan Community,Taoyuan,Nanshan
+				Shenzhen  Guangdong  518055
+				CN
+
+18-DF-26   (hex)		INGRAM MICRO SERVICES
+18DF26     (base 16)		INGRAM MICRO SERVICES
+				100 CHEMIN DE BAILLOT
+				MONTAUBAN    82000
+				FR
+
 50-56-A8   (hex)		Jollyboys Ltd
 5056A8     (base 16)		Jollyboys Ltd
 				Polttimonkatu 3
 				Tampere     33210
+				FI
+
+B0-18-31   (hex)		Apple, Inc.
+B01831     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
 				US
 
-D0-85-43   (hex)		Cisco Systems, Inc
-D08543     (base 16)		Cisco Systems, Inc
-				80 West Tasman Drive
-				San Jose  CA  94568
+D0-4D-86   (hex)		Apple, Inc.
+D04D86     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
 				US
 
-0C-7D-B0   (hex)		Motorola Mobility LLC, a Lenovo Company
-0C7DB0     (base 16)		Motorola Mobility LLC, a Lenovo Company
-				222 West Merchandise Mart Plaza
-				Chicago  IL  60654
+34-D5-09   (hex)		Genexis B.V.
+34D509     (base 16)		Genexis B.V.
+				Lodewijkstraat 1A
+				Eindhoven    5652AC
+				NL
+
+20-35-43   (hex)		Sagemcom Broadband SAS
+203543     (base 16)		Sagemcom Broadband SAS
+				250, route de l'Empereur
+				Rueil Malmaison Cedex  hauts de seine  92848
+				FR
+
+68-67-C7   (hex)		Sagemcom Broadband SAS
+6867C7     (base 16)		Sagemcom Broadband SAS
+				250, route de l'Empereur
+				Rueil Malmaison Cedex  hauts de seine  92848
+				FR
+
+84-96-90   (hex)		Murata Manufacturing Co., Ltd.
+849690     (base 16)		Murata Manufacturing Co., Ltd.
+				1-10-1, Higashikotari
+				Nagaokakyo-shi  Kyoto  617-8555
+				JP
+
+F4-B5-49   (hex)		Xiamen Yeastar Digital Technology Co., Ltd
+F4B549     (base 16)		Xiamen Yeastar Digital Technology Co., Ltd
+				Building C09, Software Park Phase III,
+				Xiamen  Fujian  361024
+				CN
+
+00-1A-01   (hex)		ICU Medical, Inc.
+001A01     (base 16)		ICU Medical, Inc.
+				4350 Rivergreen Parkway
+				Duluth  GA  30096
+				US
+
+F4-CB-E7   (hex)		Apple, Inc.
+F4CBE7     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+5C-9B-A6   (hex)		Apple, Inc.
+5C9BA6     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+38-C4-3A   (hex)		Apple, Inc.
+38C43A     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+A8-5B-D1   (hex)		INVENTEC CORPORATION
+A85BD1     (base 16)		INVENTEC CORPORATION
+				NO.88, DAZHI RD.
+				TAOYUAN    33068
+				TW
+
+2C-E0-99   (hex)		FUJIAN STAR-NET COMMUNICATION CO.,LTD
+2CE099     (base 16)		FUJIAN STAR-NET COMMUNICATION CO.,LTD
+				19-22# Building, Star-net Science Plaza, Juyuanzhou,
+				FUZHOU  FUJIAN  350002
+				CN
+
+7C-B6-8D   (hex)		Mist Systems, Inc.
+7CB68D     (base 16)		Mist Systems, Inc.
+				1601 South De Anza Blvd, Suite 248
+				Cupertino  CA  95014
+				US
+
+A4-59-D3   (hex)		IEEE Registration Authority
+A459D3     (base 16)		IEEE Registration Authority
+				445 Hoes Lane
+				Piscataway  NJ  08554
 				US
 
 C8-5C-E2   (hex)		IEEE Registration Authority
@@ -203456,12 +203789,6 @@ E80B13     (base 16)		Akib Systems Taiwan, INC
 				Nanjing    211100
 				CN
 
-00-21-0E   (hex)		Orpak Systems L.T.D.
-00210E     (base 16)		Orpak Systems L.T.D.
-				31 Lechi St
-				Bnei - Brak    51114
-				IL
-
 00-1F-F4   (hex)		Power Monitors, Inc.
 001FF4     (base 16)		Power Monitors, Inc.
 				800 North Main Street
@@ -218732,12 +219059,6 @@ CCDAB5     (base 16)		Texas Instruments
 				Dallas  TX  75243
 				US
 
-50-00-E6   (hex)		Mellanox Technologies, Inc.
-5000E6     (base 16)		Mellanox Technologies, Inc.
-				350 Oakmead Parkway, Suite 100 
-				Sunnyvale  CA  94085
-				US
-
 90-BF-D9   (hex)		Smart Innovation LLC
 90BFD9     (base 16)		Smart Innovation LLC
 				7F,Tower B,Jianxing
@@ -218750,11 +219071,17 @@ B88880     (base 16)		Shanghai Imilab Technology Co.Ltd
 				Shanghai  Shanghai  200000
 				CN
 
-38-AA-09   (hex)		Cisco Systems, Inc
-38AA09     (base 16)		Cisco Systems, Inc
-				80 West Tasman Drive
-				San Jose  CA  94568
-				US
+DC-88-A1   (hex)		ITEL MOBILE LIMITED
+DC88A1     (base 16)		ITEL MOBILE LIMITED
+				RM B3 & B4 BLOCK B, KO FAI INDUSTRIAL BUILDING  NO.7 KO FAI ROAD, YAU TONG, KLN, H.K
+				Hong Kong  KOWLOON  999077
+				HK
+
+0C-A6-4C   (hex)		Hangzhou Ezviz Software Co.,Ltd.
+0CA64C     (base 16)		Hangzhou Ezviz Software Co.,Ltd.
+				Room 302, Unit B, Building 2, 399 Danfeng Road,Binjiang District
+				Hangzhou  Zhejiang  310051
+				CN
 
 68-28-6C   (hex)		Sony Interactive Entertainment Inc.
 68286C     (base 16)		Sony Interactive Entertainment Inc.
@@ -218762,8 +219089,92 @@ B88880     (base 16)		Shanghai Imilab Technology Co.Ltd
 				Minato-ku  Tokyo  108-0075
 				JP
 
+50-00-E6   (hex)		Mellanox Technologies, Inc.
+5000E6     (base 16)		Mellanox Technologies, Inc.
+				350 Oakmead Parkway, Suite 100 
+				Sunnyvale  CA  94085
+				US
+
+38-AA-09   (hex)		Cisco Systems, Inc
+38AA09     (base 16)		Cisco Systems, Inc
+				80 West Tasman Drive
+				San Jose  CA  94568
+				US
+
 D0-AA-5F   (hex)		IEEE Registration Authority
 D0AA5F     (base 16)		IEEE Registration Authority
 				445 Hoes Lane
 				Piscataway  NJ  08554
 				US
+
+3C-BD-14   (hex)		Diebold Nixdorf
+3CBD14     (base 16)		Diebold Nixdorf
+				Wohlrabedamm 31
+				Berlin    13629
+				DE
+
+DC-32-62   (hex)		Nanjing Qinheng Microelectronics Co., Ltd.
+DC3262     (base 16)		Nanjing Qinheng Microelectronics Co., Ltd.
+				No.18, Ningshuang Road
+				 Nanjing  Jiangsu  210012
+				CN
+
+E4-FE-43   (hex)		Beijing Xiaomi Mobile Software Co., Ltd
+E4FE43     (base 16)		Beijing Xiaomi Mobile Software Co., Ltd
+				The Rainbow City Office Building, 68 Qinghe Middle Street Haidian District
+				Beijing  Beijing  100085
+				CN
+
+38-D6-E0   (hex)		TOPDON TECHNOLOGY Co.,Ltd.
+38D6E0     (base 16)		TOPDON TECHNOLOGY Co.,Ltd.
+				Nanshan
+				Shenzhen  Guangdong  518000
+				CN
+
+D8-06-F3   (hex)		Arista Networks
+D806F3     (base 16)		Arista Networks
+				5453 Great America Parkway
+				Santa Clara  CA  95054
+				US
+
+A0-9A-8E   (hex)		Apple, Inc.
+A09A8E     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+BC-9F-58   (hex)		Apple, Inc.
+BC9F58     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+78-1C-9D   (hex)		Silicon Laboratories
+781C9D     (base 16)		Silicon Laboratories
+				 400 West Cesar Chavez
+				Austin  TX  78701
+				US
+
+90-CD-E8   (hex)		Apple, Inc.
+90CDE8     (base 16)		Apple, Inc.
+				1 Infinite Loop
+				Cupertino  CA  95014
+				US
+
+70-ED-FA   (hex)		imperix Ltd
+70EDFA     (base 16)		imperix Ltd
+				Routes des Ronquos 23
+				Sion    1950
+				CH
+
+00-21-0E   (hex)		Gilbarco Inc. 
+00210E     (base 16)		Gilbarco Inc. 
+				POB 22087
+				Greensboro  NC  27420
+				US
+
+D0-AB-4A   (hex)		Telink Micro LLC
+D0AB4A     (base 16)		Telink Micro LLC
+				2975 Scott Blvd #120
+				Santa Clara    95054
+				US
diff --git a/hwdb.d/ma-medium.txt b/hwdb.d/ma-medium.txt
index 3380be11e4..ded7234753 100644
--- a/hwdb.d/ma-medium.txt
+++ b/hwdb.d/ma-medium.txt
@@ -6812,6 +6812,30 @@ A00000-AFFFFF     (base 16)		Yi Tunnel(beijing) Technology Co.,Ltd
 				Beijing    100020
 				CN
 
+D0-AA-5F   (hex)		Hangzhou Jingtang Communication Technology Co.,Ltd.
+000000-0FFFFF     (base 16)		Hangzhou Jingtang Communication Technology Co.,Ltd.
+				 8th Floor, Building 1418-42, Moganshan Road, Yuhang District,Hangzhou City, Zhejiang Province
+				Hangzhou  ZheJiang  311100
+				CN
+
+D0-AA-5F   (hex)		深圳麦源实业有限公司
+500000-5FFFFF     (base 16)		深圳麦源实业有限公司
+				广东省深圳市龙华区大浪街道澳华商务大厦1408
+				深圳  广东  518000
+				CN
+
+D0-AA-5F   (hex)		KOMER
+400000-4FFFFF     (base 16)		KOMER
+				2F. No. 583, Fujin St., Songshan Dist., Taipei City
+				Taipei  Taiwn  105
+				TW
+
+D0-AA-5F   (hex)		Shenzhen Liandian Communication Technology Co.LTD
+300000-3FFFFF     (base 16)		Shenzhen Liandian Communication Technology Co.LTD
+				1307, building A4, workshop 2, LiLang International Jewelry Industrial Park, 31 Bulan Road, xialilang community, Nanwan street, Longgang District
+				Shenzhen  Guangdong  518112
+				CN
+
 B8-4C-87   (hex)		Shenzhen Link-all Technology Co., Ltd
 300000-3FFFFF     (base 16)		Shenzhen Link-all Technology Co., Ltd
 				Floor 5th, Block 9th, Sunny Industrial Zone, Xili Town, Nanshan District, Shenzhen, China 
@@ -13121,12 +13145,36 @@ E00000-EFFFFF     (base 16)		Shenzhen Eweat Technology Co.,Ltd
 				Jiaxing  Zhejiang  314000
 				CN
 
+D0-AA-5F   (hex)		SORDIN AB
+600000-6FFFFF     (base 16)		SORDIN AB
+				Rorlaggarvagen 8
+				Varnamo  Smaland  S-331 34
+				SE
+
+D0-AA-5F   (hex)		Pacific Electronics
+C00000-CFFFFF     (base 16)		Pacific Electronics
+				No 8,  Narasigapuram street 2nd Floor Nambi trade centre, Mount Road
+				Chennai  Tamil Nadu  600002
+				IN
+
 90-88-A9   (hex)		TeeJet Technologies
 E00000-EFFFFF     (base 16)		TeeJet Technologies
 				Moelhavevej 2
 				Aabybro    9440
 				DK
 
+D0-AA-5F   (hex)		ACCEL LAB ltd.
+200000-2FFFFF     (base 16)		ACCEL LAB ltd.
+				1-8-1 Nishishinjuku Shinjuku Building 5F 
+				Shinjuku-ku  Tokyo-to  160-0023
+				JP
+
+A4-59-D3   (hex)		Shenzhen Welltech Cable Co., Ltd
+200000-2FFFFF     (base 16)		Shenzhen Welltech Cable Co., Ltd
+				3/F, Workshop 42, Xingye 1st Road, Fenghuang Community, Fuyong Subdistrict, Bao'an District
+				Shenzhen  Guangdong  518103
+				CN
+
 B8-4C-87   (hex)		Altronix , Corp
 A00000-AFFFFF     (base 16)		Altronix , Corp
 				140 58th St. Bldg A, Ste 2N
@@ -26615,6 +26663,24 @@ EC-74-CD   (hex)		Vialis BV
 				HOUTEN    3991 CJ
 				NL
 
+D0-AA-5F   (hex)		WARNER ELECTRONICS (I) PVT. LTD.
+B00000-BFFFFF     (base 16)		WARNER ELECTRONICS (I) PVT. LTD.
+				Plot No. A114/6, Five Star Industrial Area, Mumbai – Nagpur Highway, Shendra
+				Aurangabad  maharastra   431007
+				IN
+
+D0-AA-5F   (hex)		Indra Air Traffic Inc
+A00000-AFFFFF     (base 16)		Indra Air Traffic Inc
+				11300 W 89th St
+				Overland Park  KS  66214
+				US
+
+90-88-A9   (hex)		Akarui Networks Pvt Ltd
+100000-1FFFFF     (base 16)		Akarui Networks Pvt Ltd
+				B- 60 GROUND FLOOR, SECTOR - 88anand vihar
+				Noida  Uttar Pradesh  201305
+				IN
+
 90-88-A9   (hex)		Nanjing Boswell Industrial Communication Technology Co., Ltd
 900000-9FFFFF     (base 16)		Nanjing Boswell Industrial Communication Technology Co., Ltd
 				1st Floor, Building A2, China Wireless Valley, No. 9 Mozhou East Road, Jiangning District
@@ -26627,18 +26693,36 @@ EC-74-CD   (hex)		Vialis BV
 				Hangzhou  Zhejiang   310000
 				CN
 
-90-88-A9   (hex)		Akarui Networks Pvt Ltd
-100000-1FFFFF     (base 16)		Akarui Networks Pvt Ltd
-				B- 60 GROUND FLOOR, SECTOR - 88anand vihar
-				Noida  Uttar Pradesh  201305
-				IN
+D0-AA-5F   (hex)		广东美的暖通设备有限公司
+700000-7FFFFF     (base 16)		广东美的暖通设备有限公司
+				广东佛山市顺德区北滘镇蓬莱路工业大道
+				佛山    528000
+				CN
 
-D0-AA-5F   (hex)		Indra Air Traffic Inc
-A00000-AFFFFF     (base 16)		Indra Air Traffic Inc
-				11300 W 89th St
-				Overland Park  KS  66214
+D0-AA-5F   (hex)		HONGCI MONITOR INTERNATIONAL CO., LTD.
+900000-9FFFFF     (base 16)		HONGCI MONITOR INTERNATIONAL CO., LTD.
+				1 F., No. 67, Sec. 2, Zhongshan Rd., Furen Vil., Tanzi Dist
+				Taichung City    427
+				TW
+
+D0-AA-5F   (hex)		Level Up Holding Co., Inc.
+D00000-DFFFFF     (base 16)		Level Up Holding Co., Inc.
+				10210 N 25th Ave., Suite 100
+				Phoenix  AZ  85021
 				US
 
+D0-AA-5F   (hex)		WeBank Co., Ltd.
+800000-8FFFFF     (base 16)		WeBank Co., Ltd.
+				Block A, Building 7, Shenzhen Bay Technology and Ecology Park, No. 1819 Shahe West Road, Nanshan District
+				Shenzhen  Guangdong  518063
+				CN
+
+D0-AA-5F   (hex)		Xiaomi EV Technology Co., Ltd.
+100000-1FFFFF     (base 16)		Xiaomi EV Technology Co., Ltd.
+				Room 618, Floor 6, Building 5, Yard 15, Kechuang Tenth Street, Beijing Economic and Technological Development Zone, Beijing
+				Beijing  Beijing  100176
+				CN
+
 C8-5C-E2   (hex)		Fela Management AG 
 000000-0FFFFF     (base 16)		Fela Management AG 
 				Basadingerstrasse 18
@@ -33428,14 +33512,26 @@ D00000-DFFFFF     (base 16)		REMOWIRELESS COMMUNICATION INTERNATIONAL CO.,LIMITE
 				HONG KONG  HONG KONG  999077
 				HK
 
+90-88-A9   (hex)		Rikom Technologies Sdn Bhd
+C00000-CFFFFF     (base 16)		Rikom Technologies Sdn Bhd
+				Level 15, Dpulze Cyberjaya, Lingkaran, Cyber Point Timur, 63000 Selangor Malaysia
+				Cyber Point Timur  Selangor  630000
+				MY
+
 90-88-A9   (hex)		Samway Electronic SRL
 800000-8FFFFF     (base 16)		Samway Electronic SRL
 				Str Maltopol nr 16
 				Bucharest    011048
 				RO
 
-90-88-A9   (hex)		Rikom Technologies Sdn Bhd
-C00000-CFFFFF     (base 16)		Rikom Technologies Sdn Bhd
-				Level 15, Dpulze Cyberjaya, Lingkaran, Cyber Point Timur, 63000 Selangor Malaysia
-				Cyber Point Timur  Selangor  630000
-				MY
+D0-AA-5F   (hex)		MBX International Ltd.
+E00000-EFFFFF     (base 16)		MBX International Ltd.
+				RM 506A 5F HARBOUR CRYSTAL CENTRE,100 GRANVILLE ROAD TSIM SHA TSUI
+				Hongkong    XXXXX
+				HK
+
+A4-59-D3   (hex)		OREX SAI Inc.
+A00000-AFFFFF     (base 16)		OREX SAI Inc.
+				59-16 Nisshin-cho, Kawasaki-ku
+				Kawasaki  Kanagawa  210-0024
+				JP
diff --git a/hwdb.d/ma-small.txt b/hwdb.d/ma-small.txt
index df5f09b121..c2ae2f5891 100644
--- a/hwdb.d/ma-small.txt
+++ b/hwdb.d/ma-small.txt
@@ -3398,12 +3398,6 @@ A44000-A44FFF     (base 16)		FSR, INC.
 				New Taipei  Taiwan  241
 				TW
 
-70-B3-D5   (hex)		HD Vision Systems GmbH
-F18000-F18FFF     (base 16)		HD Vision Systems GmbH
-				Berliner Str. 43
-				Heidelberg    69120
-				DE
-
 70-B3-D5   (hex)		DEUTA-WERKE GmbH
 E47000-E47FFF     (base 16)		DEUTA-WERKE GmbH
 				Paffrather Str. 140
@@ -7481,6 +7475,36 @@ D04000-D04FFF     (base 16)		Plenty Unlimited Inc
 				Bergisch Gladbach  NRW  51465
 				DE
 
+8C-1F-64   (hex)		Atonarp Inc.
+42A000-42AFFF     (base 16)		Atonarp Inc.
+				PMO Shibadaimon 9F, 1-10-18 Shibadaimon  
+				Minato-ku  Tokyo  102-0018
+				JP
+
+70-B3-D5   (hex)		HD Vision Systems GmbH
+F18000-F18FFF     (base 16)		HD Vision Systems GmbH
+				Palo-Alto-Platz 11
+				Heidelberg    69124
+				DE
+
+8C-1F-64   (hex)		HD Vision Systems GmbH
+2EC000-2ECFFF     (base 16)		HD Vision Systems GmbH
+				Palo-Alto-Platz 11
+				Heidelberg    69124
+				DE
+
+8C-1F-64   (hex)		 ANSER-NET CO ., LTD
+089000-089FFF     (base 16)		 ANSER-NET CO ., LTD
+				3F., No.108, Shanghai Rd., Taoyuan Dist., Taoyuan City 330, Taiwan
+				taoyuan  taiwan  33073
+				TW
+
+8C-1F-64   (hex)		DAT Informatics Pvt Ltd
+B17000-B17FFF     (base 16)		DAT Informatics Pvt Ltd
+				Plot No 109 HPSIDC Industria Estate Davni, Baddi
+				Baddi  HIMACHAL PRADESH  173205
+				IN
+
 8C-1F-64   (hex)		Jacobs Technology, Inc.
 A98000-A98FFF     (base 16)		Jacobs Technology, Inc.
 				7765 Old Telegraph Road
@@ -15041,17 +15065,35 @@ F11000-F11FFF     (base 16)		Ottronic GmbH
 				Fohnsdorf    8753
 				AT
 
+8C-1F-64   (hex)		Point One Navigation
+4B4000-4B4FFF     (base 16)		Point One Navigation
+				231 6th St
+				San Francisco  CA  94103
+				US
+
+8C-1F-64   (hex)		SMITEC S.p.A.
+D90000-D90FFF     (base 16)		SMITEC S.p.A.
+				Via Carlo Ceresa, 10
+				San Giovanni Bianco  Bergamo  24015
+				IT
+
+8C-1F-64   (hex)		TEX COMPUTER SRL 
+249000-249FFF     (base 16)		TEX COMPUTER SRL 
+				Via O. Respighi 13
+				CATTOLICA   RIMINI   47841
+				IT
+
 8C-1F-64   (hex)		J B Electronics Corp
 421000-421FFF     (base 16)		J B Electronics Corp
 				160 Warren Pkwy, Suite 100
 				Frisco  TX  75034
 				US
 
-8C-1F-64   (hex)		Point One Navigation
-4B4000-4B4FFF     (base 16)		Point One Navigation
-				231 6th St
-				San Francisco  CA  94103
-				US
+8C-1F-64   (hex)		YUYAMA MFG Co.,Ltd
+A4A000-A4AFFF     (base 16)		YUYAMA MFG Co.,Ltd
+				1-4-30
+				MEISHINGUCHI,TOYONAKA  OSAKA  561-0841
+				JP
 
 8C-1F-64   (hex)		Vision Systems Safety Tech
 E6F000-E6FFFF     (base 16)		Vision Systems Safety Tech
@@ -22469,12 +22511,42 @@ BB0000-BB0FFF     (base 16)		Shanghai Sansi Electronic Engineering Co., Ltd.
 				Mölnlycke    SE-435 33
 				SE
 
+8C-1F-64   (hex)		Fo Xie Optoelectronics Technology Co., Ltd
+846000-846FFF     (base 16)		Fo Xie Optoelectronics Technology Co., Ltd
+				No. 115, Dazhong S. St., West Dist.,
+				Taichung  Taiwan  403025
+				TW
+
+8C-1F-64   (hex)		KST technology
+174000-174FFF     (base 16)		KST technology
+				KST B/D 4-5, Wiryeseong-daero 12-gil
+				Songpa-gu  Seoul  05636
+				KR
+
+8C-1F-64   (hex)		Pneumax Spa
+0FF000-0FFFFF     (base 16)		Pneumax Spa
+				via cascina barbellina, 10
+				Lurano  Bergamo  24050
+				IT
+
 8C-1F-64   (hex)		REO AG
 198000-198FFF     (base 16)		REO AG
 				Brühlerstr. 100
 				Solingen    42657
 				DE
 
+8C-1F-64   (hex)		KRONOTECH SRL
+5F9000-5F9FFF     (base 16)		KRONOTECH SRL
+				VIALE UNGHERIA 125
+				UDINE  ITALY/UDINE  33100
+				IT
+
+8C-1F-64   (hex)		LIMAB AB
+371000-371FFF     (base 16)		LIMAB AB
+				Almedalsvägen
+				Göteborg    41263
+				SE
+
 8C-1F-64   (hex)		Flow Power
 82B000-82BFFF     (base 16)		Flow Power
 				Suite 2, Level 3, 18 - 20 York St
@@ -30023,6 +30095,24 @@ BB8000-BB8FFF     (base 16)		ezDOOR, LLC
 				Lacombe  AB  T4L 1Y9
 				CA
 
+8C-1F-64   (hex)		OPTOKON, a.s.
+BCA000-BCAFFF     (base 16)		OPTOKON, a.s.
+				Červený Kříž 250
+				Jihlava    586 01
+				CZ
+
+8C-1F-64   (hex)		Prolife Equipamentos Médicos Ltda.
+B81000-B81FFF     (base 16)		Prolife Equipamentos Médicos Ltda.
+				732, AV. GENESIO VARGAS - RECANTO DOS IPES
+				CAMANDUCAIA  MINAS GERAIS  37650-000
+				BR
+
+8C-1F-64   (hex)		Gogo BA 
+437000-437FFF     (base 16)		Gogo BA 
+				105 Edgeview Drive
+				Broomfield  CO  80021
+				US
+
 8C-1F-64   (hex)		Mobileye
 D63000-D63FFF     (base 16)		Mobileye
 				13 Hartom st.
@@ -37432,3 +37522,33 @@ BD9000-BD9FFF     (base 16)		WATTS
 				132, Donghwagongdan-ro, Munmak-eup
 				Wonju-si  Gangwon-do  26365
 				KR
+
+8C-1F-64   (hex)		Cortical Labs Pte Ltd
+6EE000-6EEFFF     (base 16)		Cortical Labs Pte Ltd
+				20A TANJONG PAGAR ROAD
+				SINGAPORE    088443
+				SG
+
+8C-1F-64   (hex)		Horcery LLC
+E35000-E35FFF     (base 16)		Horcery LLC
+				9350 Conroy Windermere Rd
+				Windermere  FL  34786
+				US
+
+8C-1F-64   (hex)		DTI SRL
+814000-814FFF     (base 16)		DTI SRL
+				VIA PALLADIO 11
+				TAVAGNACCO  UD  33010
+				IT
+
+8C-1F-64   (hex)		CTI Intl Solutions
+7C2000-7C2FFF     (base 16)		CTI Intl Solutions
+				20 Hamagshimim
+				Petach Tikva    4934829
+				IL
+
+8C-1F-64   (hex)		新川センサテクノロジ株式会社
+E04000-E04FFF     (base 16)		新川センサテクノロジ株式会社
+				広島県東広島市吉川工業団地4-22
+				東広島市  広島県  7390153
+				JP
diff --git a/hwdb.d/usb.ids b/hwdb.d/usb.ids
index 41b367d1a0..3304de282c 100644
--- a/hwdb.d/usb.ids
+++ b/hwdb.d/usb.ids
@@ -9,8 +9,8 @@
 #	The latest version can be obtained from
 #		http://www.linux-usb.org/usb.ids
 #
-# Version: 2024.07.04
-# Date:    2024-07-04 20:34:02
+# Version: 2024.12.04
+# Date:    2024-12-04 20:34:02
 #
 
 # Vendors, devices and interfaces. Please keep sorted.
@@ -16876,7 +16876,7 @@
 	0256  Schwalm & Tate LLC pISO Raspberry Pi Hat
 	053a  Hackerspace San Salvador HSSV SAMR21-Mote
 	0cbd  Andrzej Szombierski kuku.eu.org keyboard
-	0d32  ODrive Robotics ODrive v3
+	0d32  ODrive Robotics ODrive
 	1001  InterBiometrics Hub
 	1002  InterBiometrics Relais
 	1003  InterBiometrics IBSecureCam-P
